node/ChainwebNode.hs

```diff
@@ -493,10 +492,10 @@ pkgInfoScopes =
 -- -------------------------------------------------------------------------- --
 -- main
 
--- KILLSWITCH for version 2.2
+-- KILLSWITCH for version 2.3
 --
 killSwitchDate :: Maybe String
-killSwitchDate = Just "2020-11-19T00:00:00Z"
+killSwitchDate = Just "2020-12-22T00:00:00Z"
```

Comment:
@mightybyte @slpopejoy This is just a placeholder. The final value must be still filled in.